src/anjaliS/BankingSystem.java

```diff
@@ -0,0 +1,68 @@
+package anjaliS;
+
+public class BankingSystem
+{
+	  int debitdAmtCnt;
+	  int creditAmtCnt;
+	  int balanceCnt;
+	  int currentBalance;
+	  
+	  static int totalCreditCnt;
+	  static int totalDebitCnt;
+	  static int totalBalanceCnt;
+
+	   void creditAmount(int creditAmt){
+	      currentBalance=currentBalance+creditAmt;
+		  creditAmtCnt++;
+		  totalCreditCnt++;
+      }
+	    void debitAmount(int debitAmt){  
+		  currentBalance=currentBalance-debitAmt;
```

Comment:
If condition is required. 
what if currentBalanace is less than debitAmt ?

Reply:
Yes got it, will make the changes..